src/main/java/duke/logic/parsers/commandparser/AddEventParser.java

```diff
@@ -0,0 +1,31 @@
+package duke.logic.parsers.commandparser;
+
+import duke.commons.exceptions.DukeException;
+import duke.logic.commands.AddCommand;
+import duke.logic.commands.Command;
+import duke.logic.parsers.ParserUtil;
+import duke.model.Event;
```

Comment:
👍👍
Wondering if it's possible to static the parsers, so that users don't need to new it. And to emphasize on the fact that only 1 parser is needed

Reply:
Yeah, it would be better to make the parsers static instead.